sdk/aqueduct/client.py

```diff
@@ -369,6 +371,10 @@ def publish_flow(
                 - engine: Specify where this flow should run with one of your connected integrations.
                 - k_latest_runs: Number of most-recent runs of this flow that Aqueduct should store.
                     Runs outside of this bound are deleted. Defaults to persisting all runs.
+            source_flow_id:
```

Comment:
Do we have to require to to be a UUID? Can we accept a name and convert?

Reply:
@vsreekanti Do you want 2 separate parameters:
- `source_flow_id`
- `source_flow_name`
And we use whichever one is set (using the UUID if both are present).
?

Reply:
Why don't we have the parameter be called `source_flow`:
1. We look to see if a workflow of that name exists. If it does, we use it.
2. If it doesn't, we try to convert it to a UUID. If it fails we error.
3. If it converts, we look for a workflow with that ID. If it doesn't exist we error; if it does, we use it.

Reply:
Sounds good

Reply:
Oh I didn't see Vikram's follow up comment lol, glad we came to the same name conclusion. Agreed it's just some conversion logic.

Reply:
Addressed